tests/_fixtures/v1/get_master.json

```diff
@@ -6,10 +6,10 @@
             "ip": 1694826506,
             "port": 5050,
             "pid": "master@10.0.5.101:5050",
-            "hostname": "10.0.5.101",
+            "hostname": "dcos-0",
```

Comment:
I hope it's ok that I pushed a change to fix an failing integration test assertion to match these changes 🙏

Reply:
Thank you!